packages/editor/src/store/actions.native.js

```diff
@@ -44,43 +39,3 @@ export function updateClipboard( clipboard ) {
 		clipboard,
```

Comment:
Shouldn't we remove `updateClipboard` as well here?

Reply:
Right! Good catch